### DISCUSSION TITLE
multi-account setup problem "refresh_token: Not a directory"

### DISCUSSION THREAD

**Discussion by cholzer79** [abraunegg/onedrive#2047] [Q&A] [answered]
I am trying a clean install of onedrive on Ubuntu 22.04 LTS and I run into the following problem.

Plan is to set this up for 2 onedrive accounts

Here is what I did step by step:

1. mkdir /etc/onedrive/chris/
2. wget https://raw.githubusercontent.com/abraunegg/onedrive/master/config -O /etc/onedrive/chris/config
3. nano /etc/onedrive/chris/config
4. sync_dir = "/mnt/unraid/onedrive/chris"
5. onedrive --confdir="/etc/onedrive/chris/config"

Result:  Error Message:    /etc/onedrive/chris/config/refresh_token: Not a directory

What am I doing wrong?

**Comment by abraunegg** [comment, marked as answer]
@cholzer79 
You are passing a file as a directory when using the configuration directory option `--confdir`

**Reply by cholzer79** [reply]
I am so stupid, thank you!

**Reply by abraunegg** [reply]
This specific issue fixed by https://github.com/abraunegg/onedrive/pull/2051
